.github/workflows/ci-cd.yml

```diff
@@ -97,6 +97,16 @@ jobs:
           tags: |
             ghcr.io/${{ github.repository }}-minimal:${{ github.event.release.tag_name }}
             ghcr.io/${{ github.repository }}-minimal:latest
+      - name: Build zot-exporter container image
```

Comment:
Add this to Dockerfile-arch as well? We can post the exporter for all arch then. Look at how the ARCH arg is passed in and setup.